app/src/users/views.py

```diff
@@ -294,3 +295,19 @@ def list_csv_preferences_paid(request):
         ) for p in prefs
     ]
     return csv_response(header, data_list, filename="list_csv_preferences_paid")
+
+
+@staff_member_required
+@require_http_methods(['GET'])
+def list_csv_lectures(request):
+    lectures = Lecture.objects.all()
+    header = ("Name", "Printed name", "Lecturers", "Duration", "Type",
+              "IsSponsor", "Comment")
+    data = [(
+        str(lecture.title), "", str(lecture.author), str(lecture.duration),
+        str([lt[1] for lt in LECTURE_TYPE if lt[0] == lecture.lecture_type][0]),
```

Comment:
@ref-humbold is there any built-in mechanizm for this?

Reply:
I'd go for `dict(LECTURE_TYPE)[lecture.lecture_type]`.